src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
-body {
+.body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+.card {
+  flex-direction: column;
+  padding: 32px 16px 30px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  margin: 0;
+  margin-bottom: 0;
+}
+
+.card__div-image {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px 36px;
+}
+
+.card__product {
+  width: 166px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 4px;
+  color: #060b35;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 0 0 18px;
+}
+
+.stars__stars--4 {
```

Comment:
according to [checklist ](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)point 11 every BEM block should have 1 corresponding file, please create one for card and one for stars